Project.toml

```diff
@@ -8,9 +8,13 @@ BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 Calculus = "49dc2e85-a5d0-5ad3-a950-438e2897f1b9"
 DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+DiffEqParamEstim = "1130ab10-4a5a-5621-a13d-e4788d82bd4c"
```

Comment:
You only need the Optim package here.

Reply:
Yes, sorry, I was looking for the package that has LevenbergLarquart. Ended up not using it.